### ISSUE TITLE
backend default_service MAXCONN

### ISSUE THREAD

**Issue by erikaulin** [docker/dockercloud-haproxy#78]
Hi, can you add the option to set MAXCONN environment on linked application services.

I'm trying to change to a dynamic scaling splash using dockercloud-haproxy. The old configuration using haproxy.cfg looked like this.

```
# Splash Cluster configuration
frontend http-in
    bind *:8050

    default_backend splash-cluster

    # don't apply the same limits for non-render endpoints
    acl staticfiles path_beg /_harviewer/
    acl misc path / /info /_debug /debug
    use_backend splash-misc if staticfiles or misc

backend splash-cluster
    option httpchk GET /
    balance leastconn

    # try another instance when connection is dropped
    retries 2
    option redispatch
    server splash0 splash0:8050 check maxconn 5 inter 2s fall 10 observe layer4
    server splash1 splash1:8050 check maxconn 5 inter 2s fall 10 observe layer4

backend splash-misc
    balance roundrobin
    server splash0 splash0:8050 check fall 15
    server splash1 splash1:8050 check fall 15
```

This is my current docker-compose.yml

```
version: '2'
services:
  haproxy:
      image: dockercloud/haproxy:1.2.1
      ports:
          # stats
          - "1936:1936"
          # splash
          - "8050:80"
      links:
          - splash
      volumes:
          - /var/run/docker.sock:/var/run/docker.sock
      environment:
          - BALANCE=leastconn
          - STATS_PORT=1936
          - STATS_AUTH=admin:admin

  splash:
      image: scrapinghub/splash:latest
      command: --max-timeout 3600 --slots 10 --maxrss 3000 --verbosity 1
      expose:
          - 8050
      mem_limit: 1000m
      memswap_limit: 1500m
      restart: always
      volumes:
          - ./aquarium/filters:/etc/splash/filters:ro
      environment:
          - MAXCONN=5
```

**Comment by ErikBjare** [comment]
I'm assuming (given the state of https://github.com/docker/dockercloud-haproxy/pull/79) that this can be closed and considered already fixed?

**Comment by erikaulin** [comment]
Yes, solved using release 1.5.1 and EXTRA_ROUTE_SETTINGS=maxconn 5
